pkg/virt-launcher/virtwrap/access-credentials/access_credentials.go

```diff
@@ -285,16 +285,15 @@ func (l *AccessCredentialManager) agentSetFilePermissions(domName string, filePa
 }
 
 func (l *AccessCredentialManager) agentSetUserPassword(domName string, user string, password string) error {
-
-	base64Str := base64.StdEncoding.EncodeToString([]byte(password))
-
-	cmdSetPassword := fmt.Sprintf(`{"execute":"guest-set-user-password", "arguments": {"username":"%s", "password": "%s", "crypted": false }}`, user, base64Str)
-
-	_, err := l.virConn.QemuAgentCommand(cmdSetPassword, domName)
+	domain, err := l.virConn.LookupDomainByName(domName)
 	if err != nil {
+		log.Log.Errorf("Domain lookup failed: %v", err)
 		return err
```

Comment:
Instead of logging the error, can you return a wrapped error?
```
return fmt.Errorf("Domain lookup failed: %w", err)
```

Reply:
Done.